src/Exception/EXCEPTION.md

```diff
@@ -0,0 +1,84 @@
+## Exceptions
+When you make a request via the SDK, there is a chance of request failing
+due to various reasons. When such a failure happens, an exception
+corresponding to the error occurred is raised.
+
+
+----
```

Comment:
Do we need these `----` ?

Reply:
This makes a it more beautiful :P Its like a paragraph separator. View it as an actual markdown and you shall see 😁